SU-HSDO/suhumsci
# Migrate all ECK entities and types to hs_entity

# READY FOR REVIEW

## Steps to Test
1. checkout this branch
2. `composer install -n` (requires php 8.2. do `composer update` if you're on 8.1)
3. sync a sites `blt drupal:sync --site=[your choice]`
4. verify the upgrade is successful and the site still functions.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
